.github/workflows/release.yml

```diff
@@ -54,6 +54,10 @@ jobs:
           fail_on_empty: false
       - name: Build
         run: npm run --workspace packages build --if-present
+      - name: Lint
+        env:
+          NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
Ik geloof niet dat dit hier nodig is, tenzij je tegen memory limits aan liep tijdens het uitvoeren van de linter.

Reply:
Er was een probleem aan de Github action 
https://github.com/frameless/strapi/actions/runs/5965206646/job/16182518933#step:8:27